console/db/migrations/20241205103951_2.1.0.sql

```diff
@@ -20,6 +20,35 @@ WHERE extension_name IN (
     'timescaledb'
 );
 
+
+-- Extends 20240520144338_2.0.0_initial_scheme_setup.sql#L217 with more cloud instance types
+-- Heztner price is for the region 'Geremany / Finland', other regions may vary in price.
+INSERT INTO public.cloud_instances (cloud_provider, instance_group, instance_name, cpu, ram, price_hourly, price_monthly, currency, updated_at) VALUES
+    ('hetzner', 'Small Size', 'CX22 (Shared vCPU Intel)', 2, 4, 0.0074 , 4.59, '$', '2024-12-10'),    
+    ('hetzner', 'Small Size', 'CX32 (Shared vCPU Intel)', 4, 8, 0.0127 , 7.59, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CX42 (Shared vCPU Intel)', 8, 16, 0.0304 , 18.59, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CX52 (Shared vCPU Intel)', 16, 32, 0.0611 , 36.09, '$', '2024-12-10'),
+    ('hetzner', 'Small Size', 'CPX31 (Shared vCPU AMD)', 4, 8, 0.025 , 15.59, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CPX41 (Shared vCPU AMD)', 8, 16, 0.0464 , 28.09, '$', '2024-12-10'),
+    ('hetzner', 'Medium Size', 'CPX51 (Shared vCPU AMD)', 16, 32, 0.0979 , 61.09, '$', '2024-12-10');
```

Comment:
This approach will cause errors during deployment because the server type will not be transmitted correctly. It’s better to add a shared_cpu column (boolean), which allows us to expand the API and properly display “Shared vCPU” in the UI.

@nelsonic please reverte the last commit

Reply:
Noted. So we need an "ALTER" to the table to add the new `shared_cpu` column?

Reply:
I recommend making further changes in a separate branch/PR, as there is still a lot of work to be done. I plan to merge these changes as they are for now (without the last commit).

Reply:
e.g: 
```SQL
ALTER TABLE ONLY public.cloud_instances
ADD COLUMN "shared_cpu" BOOLEAN DEFAULT FALSE;
```

Seeing as most of the other cloud providers do not _share_ the `vCPU` 💭

Reply:
Agree there is still work to be done in the UI. 👨‍🎨 👌 
But if we already have the field with the necessary data, a frontender/fullstacker (maybe me 😉)
can add via follow-up without having to create a new migration. 💭

Reply:
@vitabaks LMK if this is OK: https://github.com/vitabaks/autobase/pull/835/commits/025bbd3839a22d12dfee23a694f6f91711fef65f